lgc/patch/PatchInOutImportExport.cpp

```diff
@@ -1577,6 +1577,48 @@ Value *PatchInOutImportExport::performFsHalfInterpolation(BuilderBase &builder,
   return result;
 }
 
+// =====================================================================================================================
+// Load a specified FS parameter
```

Comment:
Would be good to mention that the function handles Flat interpolation specifically (or rename to include Flat).

Reply:
This is for both custom and flat. It actually do is simply loading a parameter. So that's why I choose this function name. The name has been differentiated as normal interpolation they are named with interpolation post-fix. And this is just used to load a parameter without interpolation. If you guys have good idea on the comment, I am happy to apply.

Reply:
updated comment: "Load a specified FS parameter (used under flat/custom interpolation)."

Reply:
Thanks!